<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="2157" started-at="2019-03-14T17:46:09Z" finished-at="2019-03-14T17:46:12Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="2157" started-at="2019-03-14T17:46:09Z" finished-at="2019-03-14T17:46:12Z">
      <class name="com.qa.tests.GetApiTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.tests.GetApiTest@7c0e2abd]" name="setUp" is-config="true" duration-ms="10" started-at="2019-03-14T23:16:09Z" finished-at="2019-03-14T23:16:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="getApiTest()[pri:0, instance:com.qa.tests.GetApiTest@7c0e2abd]" name="getApiTest" duration-ms="2072" started-at="2019-03-14T23:16:09Z" finished-at="2019-03-14T23:16:12Z">
          <exception class="org.json.JSONException">
            <message>
              <![CDATA[A JSONObject text must begin with '{' at 1 [character 2 line 1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.json.JSONException: A JSONObject text must begin with '{' at 1 [character 2 line 1]
	at org.json.JSONTokener.syntaxError(JSONTokener.java:505)
	at org.json.JSONObject.<init>(JSONObject.java:215)
	at org.json.JSONObject.<init>(JSONObject.java:399)
	at com.qa.Apiclient.Rest_Client.get(Rest_Client.java:33)
	at com.qa.tests.GetApiTest.getApiTest(GetApiTest.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.json.JSONException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getApiTest -->
      </class> <!-- com.qa.tests.GetApiTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
